pkg/core/prometheus.go

```diff
@@ -0,0 +1,86 @@
+package core
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics for monitoring service.
+var (
+	//blockHeight prometheus metric.
+	blockHeight = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Current index of persisted block",
+			Name:      "current_block_height",
+			Namespace: "neogo",
+		},
+	)
+	//persistedHeight prometheus metric.
+	persistedHeight = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Current persisted block count",
+			Name:      "current_persisted_height",
+			Namespace: "neogo",
+		},
+	)
+	//headerHeight prometheus metric.
+	headerHeight = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Current header height",
+			Name:      "current_header_height",
+			Namespace: "neogo",
+		},
+	)
+	//mempoolCount prometheus metric.
+	mempoolCount = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Mempool Count",
```

Comment:
`Overall mempool transaction count`? BTW, if that's just a sum of `mempoolUnsortedTx` and `mempoolUnverifiedTx`, maybe we can just provide those and let the user do the summing if there is a need to.

Reply:
we already have them yes, but previously you asked to add Count as well. So I added it